### ISSUE TITLE
Random collisionless objects

### ISSUE THREAD

**Issue by Xenius97** [multitheftauto/mtasa-blue#2720]
### Describe the bug

If you replace an object it could make a random collisionless object on the map.

https://streamable.com/q1z6bm
https://streamable.com/gyw52y

### Steps to reproduce

Replace an object via script (idk what ids are bugged, 7248 is)

Test resource:
[lvtestreplace.zip](https://github.com/multitheftauto/mtasa-blue/files/9399831/lvtestreplace.zip)

### Version

Multi Theft Auto v1.5.9-release-21286

### Security Policy

- [x] I have read and understood the Security Policy and this issue is not security related.

**Comment by botder** [comment]
**Model 7049 is the blurry one in the picture - you can see the outlines**
![mta-screen_2022-08-23_13-36-11](https://user-images.githubusercontent.com/38703318/186148598-39dd1258-f0a6-4362-83f3-3b9c4ef612d2.png)

**This one is, according to the world object picker, also (collision?) model 7049 and it's "losing" collision after replacement.**
![mta-screen_2022-08-23_13-37-22](https://user-images.githubusercontent.com/38703318/186148605-94648b06-01ed-43f8-b46d-f910f8a9aebe.png)

The collision-less world object is apparently linked to the one being replaced in your script and you're replacing the collision model too.

**Comment by Xenius97** [comment]
Sorry, i've said wrong. 7248 is the object what loses collision after replacing id 7049. Can't load anything to that, tried to add new collision but had no affect, looks like 7248 is totally dead after 7049 replacement. Tested on default play server too. (i've link the resource, and 2 videos)
After stopping the script and rejoining it will be again normal.

**Comment by Xenius97** [comment]
![image](https://user-images.githubusercontent.com/6568348/186164048-b809d2aa-d5d1-43b9-bf4e-701c563d994c.png)

**Comment by etrnl1337** [comment]
I don't think that they'll repair this. This is connected with LODS and also a lot of objects has the same .col file. There are a lot of objects that break when you edit them so basically its faster to repair it by yourself than waiting for fix. Good luck 4u tho.

**Comment by Xenius97** [comment]
removing && createBuilding fixed it

**Comment by TheNormalnij** [comment]
but it isn't fixed for objects
